Meta/CMake/utils.cmake

```
@@ -1,3 +1,7 @@
+
+include(${CMAKE_CURRENT_LIST_DIR}/serenity_components.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/code_generators.cmake)
```

You can just use relative paths:
```suggestion
include(serenity_components.cmake)
include(code_generators.cmake)
```

---

When I make that change, it complains that it can't find the files
```
CMake Error at Meta/CMake/utils.cmake:2 (include):
  include could not find load file:

    serenity_components.cmake
Call Stack (most recent call first):
  CMakeLists.txt:326 (include)
```
Even when utils is included like
```
include(Meta/CMake/utils.cmake)
```
Probably related to not adding Meta/CMake to the CMake include path 🤷‍♂️

---

Right, `include`d things are dynamically scoped, which applies to paths as well. This can be resolved then.